src/pattern-library/components/PlaygroundApp.tsx

```diff
@@ -1,45 +1,34 @@
 import classnames from 'classnames';
+import type { ComponentChildren } from 'preact';
 
 import { Link, LogoIcon } from '../../next';
 import Library from './Library';
 
 import { componentGroups, getRoutes } from '../routes';
+import type { CustomPlaygroundRoute, PlaygroundRoute } from '../routes';
 import { useRoute } from '../router';
 
-/**
- * @typedef {import("../routes").PlaygroundRoute} PlaygroundRoute
- * @typedef {import("../routes").CustomPlaygroundRoute} CustomPlaygroundRoute
- * @typedef {import("../routes").PlaygroundRouteGroup} PlaygroundRouteGroup
- * @typedef {import('preact').ComponentChildren} Children
- */
-
-/**
- * @typedef PlaygroundAppProps
- * @prop {string} [baseURL]
- * @prop {CustomPlaygroundRoute[]} [extraRoutes] -
- *   Local-/application-specific routes to add to this pattern library in
- *   addition to the shared/common routes
- * @prop {string} extraRoutesTitle - Title to show above any extra routes
- * provided
- */
+export type PlaygroundAppProps = {
+  baseURL?: string;
+  extraRoutes?: CustomPlaygroundRoute[];
+  extraRoutesTitle?: string;
+};
 
 /**
  * Render web content for the playground application. This includes the "frame"
  * around the page and a navigation channel, as well as the content rendered
  * by the component handling the current route.
- *
- * @param {PlaygroundAppProps} props
  */
 export default function PlaygroundApp({
   baseURL = '/ui-playground',
   extraRoutes = [],
   extraRoutesTitle,
-}) {
+}: PlaygroundAppProps) {
   const routes = getRoutes();
 
   // Put all of the custom routes into the "custom" group
   const customRoutes = extraRoutes.map(route => {
-    return /** @type PlaygroundRoute */ ({ ...route, group: 'custom' });
+    return { ...route, group: 'custom' } as PlaygroundRoute;
   });
```

Comment:
It's probably safer to add the return type to the callback, otherwise we might be returning the wrong properties here by mistake, and the type check would not warn about it:

```ts
const customRoutes = extraRoutes.map((route): PlaygroundRoute => {
  return { ...route, group: 'custom' };
});
```